experiments/xls/__init__.py

```diff
@@ -1,20 +1,72 @@
 import logging
 import os
+from typing import Dict, Callable, Union
 
 import pandas as pd
+from django.db import models
+
+from experiments.constants import *
+from experiments.models import ZPlanes, Technology, LowMagReference, TeamDirectory, Tissue, Background, Genotype, Age, \
+    ExportLocation, ArchiveLocation, Plate, Researcher, AutomatedSlide, Project, MeasurementNumber, MagBinOverlap
 
 EXCEL_TEMPLATE = os.path.join(os.path.dirname(__file__), "measurements_input.xlsx")
 
 logging.basicConfig(level=logging.INFO)
 xls_logger = logging.getLogger(__name__)
 
 
-class ExcelImporter:
-
-    """
-    Base class for importing Excel files
-    """
+class XLSImporter:
+    """Base class for importing Excel files, contains Pandas dataframe with data."""
 
     def __init__(self, file):
-        self.df = pd.read_excel(file).replace({pd.np.nan: None,
-                                               'N/A': None})
+        RESEARCHER_MAPPING = {
```

Comment:
Should this be stored in the DB? Something like a new table or an extra csv field inside Researcher?

Reply:
it's only for the old spreadsheets, once all of them are imported, this will no longer be needed